src/OrleansSQLUtils/Storage/DbConnectionFactory.cs

```diff
@@ -13,10 +14,21 @@ internal static class DbConnectionFactory
 
         private static CachedFactory GetFactory(string invariantName)
         {
+            if (String.IsNullOrWhiteSpace(invariantName))
```

Comment:
I agree, we'll definitely need to make a "style changes only" PR ;-) and make everything conform to a standard and have a resharper file also tied to the solution!